controllers/components/codeflare/codeflare.go

```diff
@@ -65,12 +57,8 @@ func (s *componentHandler) NewCRObject(dsc *dscv1.DataScienceCluster) common.Pla
 }
 
 func (s *componentHandler) Init(_ cluster.Platform) error {
-	imageParamMap := map[string]string{
-		"codeflare-operator-controller-image": "RELATED_IMAGE_ODH_CODEFLARE_OPERATOR_IMAGE",
-	}
-
-	if err := odhdeploy.ApplyParams(DefaultPath, imageParamMap); err != nil {
-		return fmt.Errorf("failed to update images on path %s: %w", DefaultPath, err)
+	if err := odhdeploy.ApplyParams(paramsPath().String(), imageParamMap); err != nil {
```

Comment:
Why does this now use params.Path(), where manifestPath() seems to be the equivalent of the previous DefaultPath?

Reply:
mh, I think I just moved everithing to use the `ManifestInfo` assuming  the `parans.env` where in the root path of one of the manifests location, but in this case it is not. 

I'll revert it

Reply:
done

Reply:
Ah right :+1: I didn't care so much that it was using a `ManifestInfo` (and then stringed), just that the path appeared to be changing here when almost everything else was more like tidying up. I can see that this is correctly fixing it back to what it was on incubation.